charts.php

```diff
@@ -151,7 +172,14 @@
yvwJDvqFZvcNvb+zrd0QoM+WJw2799+lLk06UY3PYpxysw5Lu2GdyGwDBBLw1um4JHZ/6/3tPXtx2twJAvBrdVHA0S4fyw4xUY8l3a1w3F9rRscNvqjldgyLdPqze47YN9HepoBYZ8MbjtuIL7ij291/EKDPnObAmD2ybM4DaBYSCWDW77R7F9faIZ3CYwDELlwW3vcLwCQ774DskDxfZkcJvAMBCDGtw2h7cwfKEZ3CYwDMJNfV1ScF9n9/Vmxysw5BvE4LY5voXhnc3gNoFhEP7c14EF9xVfKvyo4xUY8l3Z18+K7cngNoFhQPZr9Qa3vb+NHrqFwJCs6uC2xc3gNoFhEGJw2y3F9hSD2y70cy0w5HtpcFuxfX1k/BIQgSHZva3u4LZ3OV6BIV/c03NPsT0Z3CYwDMTz45dKEx/ctuwWgZU06eBzfX3J8QoM+WJw27kF9xXPw3mr4xUY8h3Z18OT/A2W3SKwEicdGNwmMAxEDG5bWHBfBrcJDANRcXBbvOETg9ve5HgFhnwH9/VUsT0Z3CYwDMTjfS0qGk6D2wSGAYjBbTcW21MMbos7rg1uExgGoOLgtg+1mmN1BYap84dmcBsCwwRVHNz2xja6k9zgNoEhWdw+ELcRVBvc9sm+vuZ4BYZ8v201B7fFngxuExgGIL5D8vtie4rBbT9wtAJDvkENbptDX+zr845XYMj3i1ZzcNs5zeA2gWEQYnDbn4rtKQa3fdfRCgz5DG5DYJion7R6g9vi34HBbQLDQMRT+/9SbE8xuO0IRyswDMOsfw/4C2US4pOXtYvt6Xd9neRoBYZcu/W1U8GrsYpD6ASGqRLffj2r4L7ikZq/drwCQ64Y0rZ+sT39sa/DHa3AkGtBX3sW3Nf+rd4nYgLDVJnX14Wt3jNUKn6nR2CYOvEUuE2K7anqt5IFhqmyZav5HNtvtXr3VQkMUyWexH9xX2sU21fcGf4jxysw5Dqor62K7elvreazbQSGqbJxqzkN8fhW7+l8AsPUiS+frVlsT1WfLywwTJU9+tqh2J5iQsJerd6EBIFhqqzX12kF9xUznn7jeAWG/H+I6xbbU9UplQLDVNmxr92L7Sk+Lao4Z1tgmCpr9XVuwX1d1teNjldgyBUPW9qo2J4e72uRoxUYcm3bRs/ZrSa+KPiU4xUY8sRtAHGn9GrF9nVtXz92vAJDrsV9bV5sT8/2tdDRCgy5Nuvr6IL7OrKvhx2vwJAnHh5VcdjYra3mp2ECw1TZu6/tXlOhZmZeXAPyfBtNB3jB8QoMeTbs6+SC+zqlr3sc73Cs7q9glXR2m4PBabOzg3qkyr3N4DRXMKTbta+di+3J4DSBYQDW6euHBfcVz665xfEKDLkqDk57pK8jHK3AkGtBGz1wqZq4xeEZxysw5Kk6OO3KZnCawJDu2GZwGgLDBMTgtEML7uuwvh5zvAJDnqqD027q6xLHKzDkipcQ1QanPdcMThMY0sXgtBMK7iuGwT3geAWGXOe1eoPT7mwGpwkM6b7a16eL7cngtCnkZsd6YnDa6UP8gy3/aIdXcaNkzGu6w/G6giFXxcFpDzaD01zBkO4zbcCD017D4x0MTnMFQ7IYnHZ+wX1d2tcNjldgyHViqzc47Ym+DnG0AkOubfrav+C+YnDak45XYPgfJvxw7LgNIKYDVBycdoWfHoEh12E9XptnP91/jiMag9P2dbTTz6dIK8EEH45ddXDaUX095CfHFQyJFw3jl0bziu0rBqed43hdwZAr7ijebsJXSCv7Km3peF8GpwkM/3FJ8dq+Cr8iNmijIWPVxJ7u9pPkJRK54iXE2sX2dF8bfZcHVzBM8GXCK9mlGZyGKxgmoOrgtAv6utnxCgy5YnDaBsX2FIPTDne0AkOuj7eag9PiFgeD0wSGRFUHp13V1zWOV2DIFYPTNi22p6f7OsDRCgy5tmgGpyEwTEDlwWkXO16BIVcMTtu62J5icNrezeA0gSHV/L6OL7iv2NP9jldgyBXP112r2J4MThMYBqDq4LS4HWCp4xUY8sRMo9ML7uusvm53vAJDrqqD046Z5G8w4eceIzAlxOC0Lxfc18JmcNoqyeMahiPe0D2v4L4u6+v6Sf8mQ3iqH65ghuyENvpoupIYnLbI0QoMuWJwWsX7cg5uAxic5v0ZgVmVxW0Acad0tcFp1/V1ueNdtXkPJl/c9LdFsT3F4LR9hvKH8f6MK5hVynKX7FUHp8VH0ganITCZnRm/NKo2OO22VvO5wXiJNB3Gl+zxtfnti21t6XhfBqfhCiZRPLj71IL7ij0ZnIbAJIuXEAanITDMuV3Gq9SrvvFLo+ccLwKTp+rgtIuawWkITLoY7l5tcNqjbfRdHhCYRNuPX0ZUY3AaApNs3vhlRLUbYq7u66eOF4HJFd/WNTgNgWHOxX1GFd+jiD09urJ/U3dGCwz/UnVw2q+awWkITLp4CVFxcFq8We0WZV6Re5EmZ34bPaWumthT2uA0j15wBcNIxcFpd/X1PUeLwOSKyQAVB6ft1QxOQ2BSxUyjMwvuK25xMDgNgUl2Rqs3OG1Jm/DgNASGVxYvi75ScF/xfN1nHS8rasa78oArGEBgAAQGEBhAYAAEBhAYQGAABAYQGEBgAAQGEBhAYAAEBhAYQGAABAYQGEBgAAQGEBhAYAAEBhAYQGAABAYQGACBAQQGEBgAgQEEBhAYAIEBBAYQGACBAQQGEBgAgQEEBhAYAIEBBAYQGACBAQQGEBgAgQEEBhAYAIEBBAYQGACBAQQGQGAAgQEEBkBgAIEBBAZAYACBAQQGQGAAgQEEBkBgAIEBBAZAYACBAQQGQGAAgQEEBkBgAIEBBAZAYACBAQQGQGAAgQEQGEBgAIEBEBhAYACBARAYQGAAgQEQGEBgAIEBEBhAYACBARAYQGAAgQEQGEBgAIEBEBhAYACBARAYQGAAgQEQGEBgAAQGEBhgKv1TgAEAXfcIgs3SoWAAAAAASUVORK5CYII='></img>";
```

Comment:
I did think about that, but this was quick and dirty. haha.
I think actually setting it in css might work and just calling the class.